packages/support/lib/fs.js

```diff
@@ -16,14 +18,15 @@ import log from './logger';
 import Timer from './timing';
 import pkgDir from 'pkg-dir';
 
-const mkdirAsync = B.promisify(_fs.mkdir);
-const ncpAsync = B.promisify(ncp);
+const mkdirAsync = /** @type {(filepath: string, opts: _fs.MakeDirectoryOptions|undefined) => B<void>} */(B.promisify(_fs.mkdir));
```

Comment:
can we use https://nodejs.org/api/fs.html#fspromisesmkdirpath-options, which returns a promise?

Reply:
I was just keeping things the way they already are. that said: we can replace all of the promisified `fs` methods with native ones instead of doing it ourselves, unless there's a reason to wrap them in Bluebird promises...

Reply:
there was a reason for it before.:the promisified mkdir only appeared in node 10. Now, when we set node14 to be the minimum supported version it is totally safe to drop promisifying.

Reply:
I think we're at Node.js v12 minimum now, so we can just use the builtins w/o promisifying.  I'll take care of that now.

Reply:
@mykola-mokhnach I've made those changes.